src/node_constants.cc

```diff
@@ -20,7 +20,9 @@
 #endif
 
 namespace node {
-
+#if HAVE_OPENSSL
+const char * DEFAULT_CIPHER_LIST = DEFAULT_CIPHER_LIST_CORE;
```

Comment:
The ALL CAPS name suggests it's a macro, can you change that?  Can you put it after the imports?